src/main/java/seedu/rc4hdb/ui/ResidentTableView.java

```diff
@@ -71,22 +78,22 @@ private void addColumns() {
      * Populates the columns with data from the given {@code ObservableList}.
      */
     private void populateRows() {
-        nameColumn.setCellValueFactory(new PropertyValueFactory<>("name"));
-        phoneColumn.setCellValueFactory(new PropertyValueFactory<>("phone"));
-        emailColumn.setCellValueFactory(new PropertyValueFactory<>("email"));
-        roomColumn.setCellValueFactory(new PropertyValueFactory<>("room"));
+        nameColumn.setCellValueFactory(new PropertyValueFactory<>(Name.IDENTIFIER.toLowerCase()));
+        phoneColumn.setCellValueFactory(new PropertyValueFactory<>(Phone.IDENTIFIER.toLowerCase()));
+        emailColumn.setCellValueFactory(new PropertyValueFactory<>(Email.IDENTIFIER.toLowerCase()));
+        roomColumn.setCellValueFactory(new PropertyValueFactory<>(Room.IDENTIFIER.toLowerCase()));
         indexColumn.setCellFactory(this::populateIndexColumn);
-        genderColumn.setCellValueFactory(new PropertyValueFactory<>("gender"));
-        houseColumn.setCellValueFactory(new PropertyValueFactory<>("house"));
+        genderColumn.setCellValueFactory(new PropertyValueFactory<>(Gender.IDENTIFIER.toLowerCase()));
+        houseColumn.setCellValueFactory(new PropertyValueFactory<>(House.IDENTIFIER.toLowerCase()));
         matricColumn.setCellValueFactory(new PropertyValueFactory<>("matricNumber"));
```

Comment:
I'm assuming that `matricNumber.IDENTIFER` is not working. I think you can fix this by changing the `IDENTIFIER` in `MatricNumber.java` to `MatricNumber` or `matricNumber`. Though I might be wrong.

Reply:
Yep, would work that way, however, it would change our matric number column header to MatricNumber rather than our current Matric.

Reply:
Actually, the main reason why we should not change it to `matricNumber` is due to all our other identifiers starting with uppercase letters. For e.g. the identifier for the `Gender` class is `Gender`.

As for why we should not change to `MatricNumber`, this is because we require `matricNumber`, which is not equal to `"MatricNumber".toLowerCase()".

Reply:
Also, will probably not make all the `ResidentField`s final. After some testing, it seems that it works perfectly fine, even when we check subclasses. For e.g. if we create a new class `NameExtended` which extends `Name`, as long as both `NameExtended` and `Name` have the same value, the `equals` method will return true.

Reply:
Yeap, I have no issues with this solutions 👍